src/rhub/api/lab/region.py

```diff
@@ -285,9 +299,29 @@ def get_usage(region_id, user, with_openstack_limits=None):
     if not _user_can_access_region(region, user):
         raise Forbidden("You don't have access to this region.")
 
-    return {
-        'user_quota': region.user_quota.to_dict() if region.user_quota else None,
-        'user_quota_usage': region.get_user_quota_usage(user),
-        'total_quota': region.total_quota.to_dict() if region.total_quota else None,
-        'total_quota_usage': region.get_total_quota_usage(),
-    }
+    data = region_to_usage(region, user)
+
+    return data
+
+
+def get_all_usage(keycloak: KeycloakClient, user):
+    regions = [
+        region for region in list_regions_with_permissions(keycloak, user)
+    ]
+    if not regions:
+        return problem(404, 'Not Found', 'No regions exist')
+    data = region_to_usage(regions[0], user)
+
+    def add_usage(usage1, usage2):
+        result = {}
+        for key in usage1.keys():
+            result[key] = usage1[key] + usage2[key]
```

Comment:
As `region_to_usage` may return `None` values, I think you may need something like:

```suggestion
            try:
                result[key] = usage1[key] + usage2[key]
            exception TypeError:
                if usage1[key]:
                    result[key] = usage1[key]
                elif usage2[key]:
                    result[key] = usage2[key]
                else:
                    result[key] = None
```